docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.6'
 
 services:
   client:
-    image: node:12-slim
+    image: vue-client:latest
```

Comment:
Shouldn't these be `client` and `server` to reference the docker files we have?

Reply:
no, that's a good question though. In docker-compose, if you define a `build` stage on a service, the `image` tag is actually what we want the finished/compiled image to be **named/tagged**.

Reply:
Thank you for the explanation!